src/declarative.js

```diff
@@ -107,7 +107,7 @@
     return moduleRecords[key] || (moduleRecords[key] = {
       key: key,
       dependencies: [],
-      module: new Module({}),
+      module: new Reflect.Module({}),
```

Comment:
I don't think the above matter too much? `Module` is an internally scoped reference to `Reflect.Module` and not `global.Module` I think?